src/main/java/cafeboard/Board/BoardRestController.java

```diff
@@ -0,0 +1,41 @@
+package cafeboard.Board;
+
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
+public class BoardRestController {
+    private BoardService boardService;
```

Comment:
사소한 거긴 한데 final까지 붙여 주시면 더 좋을 것 같아요
```suggestion
    private final BoardService boardService;
```